src/model/NPMResolver.ts

```diff
@@ -14,6 +14,7 @@ import {cerr, cgreen, cred, debug, sleepBusy} from "../utils";
 import {PackageVersion} from "./PackageVersion";
 import rimraf = require("rimraf");
 import Timeout = NodeJS.Timeout;
+import { execSync } from "child_process";
```

Comment:
For the consistency reasons I would go with `spawn.sync `(import spawn = require("cross-spawn");)